### PR TITLE
Ignore =over / =back directives

### PR DESCRIPTION
**Pull request by jmaslak**
=over / =back were displaying as if they were headings, with the =over number being a heading.

I.E. =over 4 becomes:

over
====

4

And =back becomes:

back
====

This just ignores both directives for lack of a clear better thing to do.  I did wrap the in Debug calls like the Config and other directives.  Let me know if you have a different preference.

### CODE REVIEW

**Comment by softmoth** [comment]
Joelle, thanks for contributing this. I am conflicted about this change ... it gets at some problems w/ current pod handling (well, current last time I looked into this). As I understand [the spec][], and would like things to behave, `=over` and `=back` should be errors: `Typenames that are entirely lowercase (for example: C<=begin head1>) or entirely uppercase (for example: C<=begin SYNOPSIS>) are reserved.`
[the spec]: https://design.perl6.org/S26.html#Delimited_blocks

However, the pod implementations have treated unrecognized typenames as headings and people have been relying on that in the wild. In this particular case, they're obviously Perl5-isms, and to be nice we should probably warn the user and then ignore them as you've chosen to do. But I'd rather not special-case things unnecessarily, and I'm leaning towards leaving the behavior as it is. Users will notice the bad formatting of `=over` and `=back` and be able to remove them from their pod.

I'd chosen to debug on 'config' and 'nested' because they're spec'd but I'm not sure they exist in the wild or should be implemented, and want feedback and/or implementation should someone be able to use them. I don't think that's the same case here; this is just a plain user error.

tl;dr: I'm leaning towards not doing anything here, but open to being persuaded otherwise!

**Comment by jmaslak** [comment]
I'm pretty new to Perl 6 - so I wasn't aware of the changes to POD in Perl 6 (Doh!).  I've got some reading to do to get my POD up to date.  Thanks for the effort in responding to this, and I agree completely that this PR should be rejected.
